ldes-consumer/app.py

```
@@ -0,0 +1,591 @@
+    Load LDES feeds configuration from YAML file
+
+    Args:
+        config_path: Path to the ldes-feeds.yaml file
+
+    Returns:
+        Dictionary containing the feeds configuration
+    """
+    try:
+        with open(config_path, "r") as f:
+            config = yaml.safe_load(f)
+        return config
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)